UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/FileShareService.cs

```diff
@@ -30,8 +29,8 @@ public async Task<IResult<BatchSearchResponse>> FSSBatchSearchAsync(string searc
 
                 _logger.LogInformation(EventIds.FSSBatchSearchResponseStarted.ToEventId(), "Maritime safety information request for FSS to get NM batch search response started for correlationId:{correlationId} and searchQuery:{searchQuery}", correlationId, searchQuery);
 
-                FileShareApiClient fileShareApi = new(_httpClientFactory, _fileShareServiceConfig.Value.BaseUrl, accessToken);
-                result = await fileShareApi.Search(searchQuery, _fileShareServiceConfig.Value.PageSize, _fileShareServiceConfig.Value.Start, CancellationToken.None);
+                result = await fileShareApiClient.Search(searchQuery, _fileShareServiceConfig.Value.PageSize, _fileShareServiceConfig.Value.Start, CancellationToken.None);
```

Comment:
result can be declared and returned from try block:

```suggestion
             IResult<BatchSearchResponse> result = await fileShareApiClient.Search(searchQuery, _fileShareServiceConfig.Value.PageSize, _fileShareServiceConfig.Value.Start, CancellationToken.None);
```

Reply:
Implemented suggestion